server/@types/express/index.d.ts

```diff
@@ -44,7 +38,6 @@ declare module 'express-session' {
     prisonerListSortOptions: string
     pageFlowHistory: PageFlow
     pageFlowQueue: PageFlow
-    prisonerFunctionalSkills: FunctionalSkills
```

Comment:
The middleware (below 👇 ) that retrieves functional skills from Curious now does not store the data on the session, hence we can remove it from here 👍